easybuild/easyconfigs/p/PyOpenGL/PyOpenGL-3.1.5-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,44 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
+# License::
+#
+# Notes::
+##
+
+easyblock = 'PythonPackage'
+
+name = 'PyOpenGL'
+version = '3.1.5'
+
+homepage = 'http://pyopengl.sourceforge.net'
+description = """PyOpenGL is the most common cross platform Python binding to OpenGL and related APIs."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['4107ba0d0390da5766a08c242cf0cf3404c377ed293c5f6d701e457c57ba3424']
+
+multi_deps = {'Python': ['3.7.4', '2.7.16']}
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('bzip2', '1.0.8'),
+]
+
+dependencies = [
+    ('Mesa', '19.1.7'),
+]
+
+download_dep_fail = True
+use_pip = True
```

Comment:
Aren't some of these redundant now with recent changes to the `easyblock`, or is that just for `PythonBundle`?

Reply:
what changes do you refer to?

Reply:
@verdurin ping

Reply:
@lexming I thought that at least some of those parameters were now the default. Could be wrong.

Reply:
@verdurin AFAIK there have been no changes to `PythonPackages` in that regard.